pkgs/os-specific/linux/freeipa/default.nix

```
@@ -0,0 +1,97 @@
+{ stdenv, lib, fetchurl, writeText, pkgconfig, autoconf, automake, kerberos,
+  openldap, popt, sasl, curl, xmlrpc_c, ding-libs, p11-kit, gettext, nspr, nss,
+  dirsrv, svrcore, libuuid, talloc, tevent, samba, libunistring, libverto,
+  libpwquality, systemd, python3, bind, pyhbac, jre, rhino, lesscpy, jansson,
+  git }:
```

```suggestion
{ stdenv, lib, fetchurl, writeText, pkgconfig, autoconf, automake, kerberos
, openldap, popt, sasl, curl, xmlrpc_c, ding-libs, p11-kit, gettext, nspr, nss
, dirsrv, svrcore, libuuid, talloc, tevent, samba, libunistring, libverto
, libpwquality, systemd, python3, bind, pyhbac, jre, rhino, lesscpy, jansson
}:
```
Please cleanup other unused inputs. deadnix can help with that.